openedx/features/marketplace/tests/factories.py

```diff
@@ -8,7 +11,7 @@
 class ChallengeFactory(OrganizationBaseFactory, LocationFactory):
     """Factory for MarketplaceRequest model. It contains fake data or sub-factories for all mandatory fields"""
 
-    class Meta:
+    class Meta(object):
```

Comment:
Any specific reason for this change?

Reply:
Yes, it was highlighted in pylint report (you can see the complete [report on JIRA ](https://philanthropyu.atlassian.net/browse/LP-2285?focusedCommentId=18884)story)

`openedx/features/marketplace/tests/factories.py:11: [C1001(old-style-class), ChallengeFactory.Meta] Old-style class defined.`